docs/node/run-your-node/paratime-client-node.mdx

```diff
@@ -216,6 +216,38 @@ accessible on the internet (for `TCP` and `UDP` traffic).
 
 :::
 
+## Configuring TEE Paratime Client Node
```

Comment:
I don't see an explanation why or when one would want to do this. We should add it and maybe reword when this would be required -- probably something along the lines that this is needed if one wants a node that can issue queries that access confidential data.